ec/armada/data_source_armada_armada.go

```diff
@@ -0,0 +1,23 @@
+package armada
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+// DataSourceArmadaArmada returns the data source resource for an Armada.
+func DataSourceArmadaArmada() *schema.Resource {
```

Comment:
Why is "Armada" present twice in this func/file name? Since everything is about Armada, can't we just drop the first Armada in all these files/funcs? Or is it a naming scheme from Terraform we can't avoid?

Reply:
It is armada the group and armada the object, so the duplication is on purpose.